runtime/near-vm-runner/src/tests/test_builder.rs

```diff
@@ -94,8 +94,6 @@ impl TestBuilder {
         self
     }
 
-    // We only test trapping tests on Wasmer, as of version 0.17, when tests executed in parallel,
-    // Wasmer signal handlers may catch signals thrown from the Wasmtime, and produce fake failing tests.
     pub(crate) fn skip_wasmtime(mut self) -> Self {
```

Comment:
Are we in a position to remove this function altogether?

Reply:
Almost there, first #1987 will have to be merged

Reply:
Now that #1987 is merged, we should be able to do that indeed.

We can also now remove `only_wasmtime` and `skip_near_vm`